tests/TestCase/ORM/BehaviorTest.php

```diff
@@ -50,6 +50,13 @@ public function afterRules()
     public function buildRules()
     {
     }
+
+    /**
+     * Test for event bindings.
+     */
+    public function afterSaveCommit()
+    {
+    }
```

Comment:
Well, I figured that since the test isn't testing for all possible callbacks, just having one should be sufficient. I can add afterDeleteCommit if you want.